mne/preprocessing/ica.py

```diff
@@ -741,7 +745,7 @@ def score_sources(self, inst, target=None, score_func='pearsonr',
             Callable taking as arguments either two input arrays
             (e.g. Pearson correlation) or one input
             array (e. g. skewness) and returns a float. For convenience the
-            most common score_funcs are available via string labels: Currently,
+            most common score_funcs are available via string labels_: Currently,
```

Comment:
it's an attribtue we set later. Following the sklearn convention for this we let it terminate with an underscore.